components/gitpod-db/go/workspace_instance.go

```diff
@@ -74,16 +74,16 @@ func FindStoppedWorkspaceInstancesInRange(ctx context.Context, conn *gorm.DB, fr
 	return instances, nil
 }
 
-// FindRunningWorkspaceInstances finds WorkspaceInstanceForUsage that are running at the point in time the querty is executed.
+// FindRunningWorkspaceInstances finds WorkspaceInstanceForUsage that are running at the point in time the query is executed.
 func FindRunningWorkspaceInstances(ctx context.Context, conn *gorm.DB) ([]WorkspaceInstanceForUsage, error) {
 	var instances []WorkspaceInstanceForUsage
 	var instancesInBatch []WorkspaceInstanceForUsage
 
 	tx := queryWorkspaceInstanceForUsage(ctx, conn).
 		Where("wsi.stoppingTime = ?", "").
 		Where("wsi.usageAttributionId != ?", "").
-		// We cannot guarantee data quality before this date
-		Where("wsi.startedTime > ?", TimeToISO8601(time.Date(2022, 8, 1, 0, 0, 0, 0, time.UTC))).
+		// We are only interested in instances that have been started within the last 10 days.
+		Where("wsi.startedTime > ?", TimeToISO8601(time.Now().Add(-10*24*time.Hour))).
```

Comment:
For testing, it may be useful to inject the `time.now()` such that you get to control it

Reply:
Does this already ensure the time is UTC?

Reply:
we are talking days here. I don't think such fine-grained control is necessary and I wouldn't want to open up the API for that

Reply:
> Does this already ensure the time is UTC?

Doesn't really matter much in this context, but `TimeToISO8601` does the UTC conversion also